Parse/src/test/java/com/parse/NetworkObjectControllerTest.java

```diff
@@ -36,6 +40,20 @@
 @Config(constants = BuildConfig.class, sdk = 21)
 public class NetworkObjectControllerTest {
 
+  @Before
+  public void setUp() {
```

Comment:
I think you can just add `throws MalformedURLException` to the method signature instead.

This should also enforce that the test fails if there's an issue.